protocol-designer/src/components/StepEditForm/fields/PathField/getDisabledPathMap.ts

```diff
@@ -1,30 +1,22 @@
-// @flow
 import { i18n } from '../../../../localization'
 import { getWellRatio } from '../../../../steplist/utils'
 import { getPipetteCapacity } from '../../../../pipettes/pipetteData'
 import {
   volumeInCapacityForMultiDispense,
   volumeInCapacityForMultiAspirate,
 } from '../../../../steplist/formLevel/handleFormChange/utils'
-
-import type {
-  ChangeTipOptions,
-  PipetteEntities,
-} from '@opentrons/step-generation'
-import type { PathOption } from '../../../../form-types'
-
-export type DisabledPathMap = { [PathOption]: string } | null
-
-export type ValuesForPath = {|
-  aspirate_airGap_checkbox: ?boolean,
-  aspirate_airGap_volume: ?string,
-  aspirate_wells: ?Array<string>,
-  changeTip: ChangeTipOptions,
-  dispense_wells: ?Array<string>,
-  pipette: ?string,
-  volume: ?string,
-|}
-
+import { ChangeTipOptions, PipetteEntities } from '@opentrons/step-generation'
+import { PathOption } from '../../../../form-types'
+export type DisabledPathMap = Partial<Record<PathOption, string>> | null
+export interface ValuesForPath {
+  aspirate_airGap_checkbox?: boolean | null
+  aspirate_airGap_volume?: string | null
+  aspirate_wells?: string[] | null
+  changeTip: ChangeTipOptions
+  dispense_wells: string[] | null | undefined
```

Comment:
same same, but for consistency 
```suggestion
  dispense_wells?: string[] | null
```